app/models/tip.rb

```diff
@@ -1,5 +1,5 @@
 class Tip < ApplicationRecord
-  belongs_to :users
+  belongs_to :user, optional: true
```

Comment:
consider using `u.tips.create(...)` so you can remove `optional: true`. When would a tip *not* have an author?